src/libs/antares/study/parameters.cpp

```diff
@@ -539,6 +539,10 @@ static bool SGDIntLoadFamily_General(Parameters& d,
     {
         return value.to<uint>(d.nbTimeSeriesSolar);
     }
+    if (key == "nbtimeserieslinks")
```

Comment:
We know generating TS for links should be fully separated from the Antares solver.
What is true for the code is true for data as well.
For now data required generating TS for links are mixed data with the study data.
Here is a good example : **nbtimeserieslinks** is still held by **generaldata.ini**. Same for **seed_links**.
Eventually, we should separate data for TS generation form study data.

Reply:
See your PR @guilpier-code